Reference/Routing/surface-controllers.md

```diff
@@ -84,15 +83,53 @@ namespace name.Core.Controllers
 }
 ```
 
-In the above, I've specified that I'd like my MyController to belong to the MVC Area called 'SuperAwesomeAnalytics'. Perhaps it is obvious but if you are creating a package that contains many SurfaceControllers then you should most definitely ensure that all of your controllers are routed through the same MVC Area.
+In the above, the surface controller will belong to the MVC area called 'SuperAwesomeAnalytics'. Perhaps it is obvious, but if you are creating a package that contains many surface controllers, then you should most definitely ensure that all of your controllers are routed through the same MVC area.
 
 #### Routing for plugin based controllers
 
 All plugin based controllers get routed to:
 
     /umbraco/{areaname}/{controllername}/{action}/{id}
 
-Since they get routed via an MVC Area your views should be placed in the following folder:
+Since they get routed via an MVC area, your views should be placed in the following folder:
 
 * `~/App_Plugins/{areaname}/Views/{controllername}/`
 * `~/App_Plugins/{areaname}/Views/Shared/`
+
+#### Protecting surface controller routes
+
+If you only want a surface controller action to be available when it's used within an Umbraco form and not from the auto-routed URL, you can add the `[ValidateUmbracoFormRouteString]` attribute to the action method. This can be especially useful for plugin based controllers, as this makes sure the actions can only be activated whenever it's actually used within the website.
```

Comment:
```suggestion
If you only want a surface controller action to be available when it's used within an Umbraco form and not from the auto-routed URL, you can add the `[ValidateUmbracoFormRouteString]` attribute to the action method. This can be especially useful for plugin based controllers, as this makes sure the actions can only be activated whenever it's used within the website.
```